Source/Details/ASRangeController.mm

```diff
@@ -211,40 +208,13 @@ - (void)_updateVisibleNodeIndexPaths
   
   ASElementMap *map = [_dataSource elementMapForRangeController:self];
 
-  // TODO: Consider if we need to use this codepath, or can rely on something more similar to the data & display ranges
-  // Example: ... = [_layoutController indexPathsForScrolling:scrollDirection rangeType:ASLayoutRangeTypeVisible];
-  NSArray<NSIndexPath *> *visibleNodePaths = [_dataSource visibleNodeIndexPathsForRangeController:self];
-  ASWeakSet *newVisibleNodes = [[ASWeakSet alloc] init];
-
-  if (visibleNodePaths.count == 0) { // if we don't have any visibleNodes currently (scrolled before or after content)...
-    [self _setVisibleNodes:newVisibleNodes];
-    return; // don't do anything for this update, but leave _rangeIsValid == NO to make sure we update it later
-  }
-  ASProfilingSignpostStart(1, self);
```

Comment:
It is of course important that the signpost is started before the expensive indexPath gathering work, so this should be moved back.